### PR TITLE
Update SDK performance content

### PR DESCRIPTION
**Pull request by PeloWriter**
The current Included Instrumentation pages aren't quite capturing what's happening accurately with that title. As a result, we're renaming to be more accurately "automatic instrumentation" and also building out a structure that supports Instrumentation as a concept, where users can then review whether the SDK supports automatic instrumentation, custom instrumentation, or both.

### CODE REVIEW

File: src/docs/product/performance/trace-view.mdx [hunk 1]
```diff
@@ -39,4 +39,4 @@ Broken subtraces may be caused by:
 
 ### Multiple Roots
 
-Each trace ID should have only one root, a transaction without any parents. Included instrumentation should prevent multiple roots; however, if the trace ID of your transactions are being set using custom instrumentation, you may encounter multiple roots.
+Each trace ID should have only one root, a transaction without any parents. Automatic instrumentation should prevent multiple roots; however, if the trace ID of your transactions are being set using custom instrumentation, you may encounter multiple roots.
```
Comment (lobsterkatie) [inline src/docs/product/performance/trace-view.mdx:42]:
```suggestion
Each trace ID should have only one root, a transaction without any parents. Automatic instrumentation should prevent multiple roots; however, if the trace ID of your transactions is being set using custom instrumentation, you may encounter multiple roots.
```

File: src/gatsby/createPages/createPlatformPages.ts [hunk 2]
```diff
@@ -61,7 +61,7 @@ const isGuideRoot = (node: FileNode): boolean => {
  * Used to test if a parent page exists for a child. This is used to avoid creation
  * of children when a parent is hidden.
  *
- * for example, if /performance/ is notSupported, there's no need to mark /performance/custom-instrumentation/
+ * for example, if /performance/ is notSupported, there's no need to mark /performance/instrumentation/custom-instrumentation/
```
Comment (lobsterkatie) [inline src/gatsby/createPages/createPlatformPages.ts:64]:
```suggestion
 * for example, if /performance/ is notSupported, there's no need to mark /performance/instrumentation/ or /performance/instrumentation/custom-instrumentation/
```

File: src/includes/getting-started-config/javascript.ember.mdx [hunk 3]
```diff
@@ -34,6 +34,6 @@ ENV["@sentry/ember"] = {
 
 <Note>
 
-This SDK uses Ember configuration conventions to manage its included instrumentation and other Sentry options, this additional configuration can be found under <PlatformLink to="/configuration/ember-options/">Ember options</PlatformLink>.
+This SDK uses Ember configuration conventions to manage its automatic instrumentation and other Sentry options, this additional configuration can be found under <PlatformLink to="/configuration/ember-options/">Ember options</PlatformLink>.
```
Comment (lobsterkatie) [inline src/includes/getting-started-config/javascript.ember.mdx:37]:
```suggestion
This SDK uses Ember configuration conventions to manage its automatic instrumentation and other Sentry options. This additional configuration can be found under <PlatformLink to="/configuration/ember-options/">Ember options</PlatformLink>.
```

File: src/platforms/common/performance/connect-services.mdx [hunk 4]
```diff
@@ -16,9 +16,9 @@ For traces that begin in JavaScript, any requests it makes (and any requests you
 
 All of Sentry's tracing-related integrations (`BrowserTracing`, `Http`, and `Express`) either generate or pick up and propagate the trace header automatically as appropriate for all transactions and spans that they generate.
 
-The JavaScript SDK will only attach the trace header to outgoing HTTP requests whose destination contains a string in or matches a regex in the <PlatformLink to="/performance/included-instrumentation/#tracingorigins">tracingOrigin</PlatformLink> list.
+The JavaScript SDK will only attach the trace header to outgoing HTTP requests whose destination contains a string in or matches a regex in the <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#tracingorigins">tracingOrigin</PlatformLink> list.
```
Comment (lobsterkatie) [inline src/platforms/common/performance/connect-services.mdx:19]:
```suggestion
The JavaScript SDK will only attach the trace header to outgoing HTTP requests whose destination contains a string in or matches a regex in the <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#tracingorigins">tracingOrigins</PlatformLink> list.
```

File: src/platforms/common/performance/index.mdx [hunk 5]
```diff
@@ -63,13 +63,13 @@ Learn more about how the options work in <PlatformLink to="/configuration/sampli
 
 <PlatformSection supported={["react-native", "java.spring", "java.spring-boot", "android"]} notSupported={["javascript"]}>
 
-Test out tracing by adding our <PlatformLink to="/performance/included-instrumentation/">included instrumentation</PlatformLink> or by starting and finishing a transaction using <PlatformLink to="/performance/custom-instrumentation/">custom instrumentation</PlatformLink>.
+Test out tracing by adding our <PlatformLink to="/performance/instrumentation/automatic-instrumentation/">automatic instrumentation</PlatformLink> or by starting and finishing a transaction using <PlatformLink to="/performance/instrumentation/custom-instrumentation/">custom instrumentation</PlatformLink>.
```
Comment (lobsterkatie) [inline src/platforms/common/performance/index.mdx:66]:
```suggestion
Test out tracing using our <PlatformLink to="/performance/instrumentation/automatic-instrumentation/">automatic instrumentation</PlatformLink> or by starting and finishing a transaction using <PlatformLink to="/performance/instrumentation/custom-instrumentation/">custom instrumentation</PlatformLink>.
```

**Review by lobsterkatie — APPROVED** [review]
Just some proofreading. Otherwise LGTM.

File: src/docs/product/performance/trace-view.mdx (hunk 1 above)
Comment (PeloWriter) [reply src/docs/product/performance/trace-view.mdx:42]: Thanks for the catch.

**Pull request merged**